src/cls/IPM/Utils/Module.cls

```diff
@@ -419,6 +418,13 @@ ClassMethod LoadModuleFromArchive(pModuleName As %String, pModuleVersion As %Str
 		If $$$ISERR(tSC) {
 			Quit
 		}
+		// Delete directory at the end of the run
```

Comment:
If something goes horribly wrong, we won't delete the directory.

I think this is a feature.